portfolio/src/main/webapp/script.js

```diff
@@ -106,22 +106,59 @@ function createComment(element) {
   liComment.innerText = element.key + ": " + element.value;
   liComment.className = "commentli";
   liComment.id = element.key + element.value;
-  const deleteComment = document.createElement('button');
-  deleteComment.innerText = "X";
-  deleteComment.className = "singleDelete";
+  const deleteComment = addDeleteButton();
   liComment.appendChild(deleteComment);
   deleteComment.addEventListener("click", function() {
     deleteSingleComment(liComment.id);
   });
+  const reactToComment = addReactionButton();
+  liComment.appendChild(reactToComment);
+  const reactionContainer = document.createElement('div');
+  reactionContainer.className = "reactionContainer";
+  liComment.appendChild(reactionContainer);
+  reactToComment.addEventListener("change", function() {
+    const reaction = addReaction(reactToComment.options[reactToComment.selectedIndex].innerHTML);
+    reactionContainer.appendChild(reaction);
+  });
   return liComment;
 }
```

Comment:
This could use more breathing room, eg.

```
  const deleteComment = addDeleteButton();
  liComment.appendChild(deleteComment);

  deleteComment.addEventListener("click", function() {
    deleteSingleComment(liComment.id);
  });

  const reactToComment = addReactionButton();
  liComment.appendChild(reactToComment);

  const reactionContainer = document.createElement('div');
  reactionContainer.className = "reactionContainer";
  liComment.appendChild(reactionContainer);

  reactToComment.addEventListener("change", function() {
    const reaction = addReaction(reactToComment.options[reactToComment.selectedIndex].innerHTML);
    reactionContainer.appendChild(reaction);
  });

  return liComment;
}
```

It's easier to read if you bunch up your lines of code in by which component they deal with in your DOM rather than have it be a large unbroken set of lines of code that are otherwise hard to distinguish from one-another.